### PR TITLE
[stable/rabbitmq-ha] allow setting different topology keys for anti affinity

### PR DESCRIPTION
**Pull request by faraazkhan**
#### What this PR does / why we need it:
Allows chart users to set custom topology keys for anti affinity, in some cases it makes more sense to use the failure domain as the topology key rather than the hostname, this PR makes that possible. The default is still set to hostname, so users installing with default values should see no change in behavior. 

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x ] Chart Version bumped
- [x ] Variables are documented in the README.md
- [ x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/rabbitmq-ha/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.7.12
-version: 1.25.0
+version: 1.25.1
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/Chart.yaml]: New feature -> `1.26.0`. Other than that it looks good.

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: stable/rabbitmq-ha/Chart.yaml (hunk 1 above)
Comment (faraazkhan) [reply stable/rabbitmq-ha/Chart.yaml]: Done.

**Comment by steven-sheehy** [comment]
/lgtm
Thanks!

**Pull request merged**